lld/test/COFF/debug_dir_magic_strings_from_section_pgi.s

```diff
@@ -0,0 +1,16 @@
+// This test checks if lld puts magic string "PGI" when an object files contains
+// .pgi section.
+
+// RUN: llvm-mc -filetype=obj -triple=x86_64-pc-windows %s -o %t.main_x86.obj
+// RUN: llvm-mc -filetype=obj -triple=aarch64-pc-windows %s -o %t.main_aarch.obj
```

Comment:
This test requires x86 and aarch64 to be available - add suitable `REQUIRES:` lines. (While they test the same thing, they have different spellings between the clang and lld test directories, iirc, so check other surrounding tests for how it is written.)